google-cloud-tools-plugin/build.gradle

```diff
@@ -61,6 +61,7 @@ dependencies {
         exclude group: 'com.google.guava', module: 'guava'
         exclude group: 'com.google.oauth-client', module: 'google-oauth-client'
     }
+    compile 'com.google.apis:google-api-services-servicemanagement:v1-rev374-1.22.0'
```

Comment:
Just wondering, is it google-api-services-* practice to specify very exact revisions and versions in dependencies?

Reply:
I guess so. I don't really have more info on that other than this is the way they specify the versions of their binaries.